packages/es-components/.babelrc

```diff
@@ -1,7 +1,7 @@
 {
   "plugins": [
-    "transform-runtime",
-    "transform-export-extensions"
```

Comment:
Something that I want to do is migrate this `.babelrc` file to the newly introduced `babel.config.js` or `.babelrc.js` formats. This will allows us to clean up the redundant configs. I haven't verified if the below works very well, but it described the general idea.

```js
// .babelrc.js
const wtwConfig = {
  extractFormatMessage: false,
  transformFormatMessage: false
};

if (process.env['BABEL_ENV'] === 'es6') {
  wtwConfig.env = { modules: false };
}

module.exports = {
  presets: [ ['wtw-im', wtwConfig] ]
  plugins: [ '@babel/plugin-transform-runtime', '@babel/plugin-proposal-export-default-from' ]
};
```

Reply:
I think this would be good, but I'm not sure it's work we should do as part of this PR. This is just about updating the dependencies and getting everything working with them. I think we should push improvements on top of that to another PR.

Reply:
I don't see a reason to push off cleaning up our messy configs when we're already in the area. That's been one of the many reasons to update and it's very minimal effort.